browser/brave_drm_tab_helper.cc

```diff
@@ -5,15 +5,59 @@
 
 #include "brave/browser/brave_drm_tab_helper.h"
 
+#include <algorithm>
+#include <vector>
+
 #include "brave/browser/widevine/widevine_utils.h"
 #include "brave/common/pref_names.h"
+#include "chrome/browser/browser_process_impl.h"
 #include "chrome/browser/profiles/profile.h"
-#include "chrome/browser/profiles/profile_manager.h"
+#include "chrome/browser/subresource_filter/chrome_subresource_filter_client.h"
+#include "chrome/browser/ui/browser_finder.h"
+#include "chrome/browser/ui/tabs/tab_strip_model.h"
 #include "components/prefs/pref_service.h"
 #include "content/public/browser/navigation_handle.h"
 
+using component_updater::ComponentUpdateService;
+
+namespace {
+
+// Copied from widevine_cdm_component_installer.cc.
+// There is no shared constant value.
+constexpr char kWidevineComponentId[] = "oimompecagnajdejgnnjijobebaeigek";
+
+bool IsAlreadyRegistered(ComponentUpdateService* cus) {
+  std::vector<std::string> component_ids;
+  component_ids = cus->GetComponentIDs();
+  return std::find(component_ids.begin(),
+                   component_ids.end(),
+                   kWidevineComponentId) != component_ids.end();
+}
+
+content::WebContents* GetActiveWebContents() {
+  if (Browser* browser = chrome::FindLastActive())
+    return browser->tab_strip_model()->GetActiveWebContents();
+  return nullptr;
+}
+
+void ReloadingIfThisTabIsActive(content::WebContents* web_contents) {
+  if (GetActiveWebContents() == web_contents) {
+    ChromeSubresourceFilterClient::FromWebContents(web_contents)
```

Comment:
not sure I understand why do we use subresource filter here?... this can be done either via `web_contents()->GetController().Reload()` or using `browser_commands.h`

Reply:
Oh, I just c&p previously used api. changed to use controller. :)